addons/portal/__manifest__.py

```diff
@@ -25,6 +25,7 @@
     ],
     'qweb': [
         'static/src/xml/portal_chatter.xml',
+        'static/src/xml/portal_name_and_signature.xml',
```

Comment:
This would slow down the backend for nothing, to remove :wink:

Reply:
What does it do exactly? Why would it slow it down and why are the others qweb needed there?

Btw the widget is used in the backend too when doing "sign directly".

Reply:
Don't know about the other files, they can stay there for now anyway.

What happens is:
- The webclient is started
- All the files in the 'qweb' section of all the relevant \_\_manifest\_\_ are loaded in one RPC call
- ...
- You instantiate your widget at some point, the `xmlDependencies` are read and your xml file is loaded...

So bad because:
- The webclient took extra time (ok really nothing but anyways) to load your xml file content while the portal dialog will maybe never be used or maybe a lot later
- The portal dialog reloads the qweb templates for nothing

The good way in this case is using `xmlDependencies` only as we want to only load the related xml if and once a dialog is instantiated.

Note: in the frontend, only `xmlDependencies` is used.
Note 2: the `xmlDependencies` system is quite new, that's why there probably are a lot of cases where we should use it and we do not

Reply:
Thank you for the details, I removed the line.